src/App.tsx

```diff
@@ -9,62 +9,39 @@ import { ScenarioPage } from "./react-modules/pages/ScenarioPage";
 import { DEFAULT_JSON } from "./DEFAULT_JSON";
 import { JSONPreview } from "./react-modules/JSONPreview";
 import { Footer } from "./react-modules/Footer";
+import { Persona, ShowcaseJSON } from "./types";
+
+export function updateJSONWithImmer(
```

Comment:
Should we move this to a lib/helper location?

Reply:
is no longer used